src/features/emails/components/EmailEditor/EmailSettings/PreviewTab.tsx

```diff
@@ -35,6 +37,16 @@ const PreviewTab: FC = () => {
 
   return (
     <Box display="flex" flexDirection="column" gap={2} padding={2}>
+      <Typography>
+        <Msg
+          id={
+            messageIds.editor.settings.tabs.preview.previewInBrowserInstructions
+          }
+        />
+      </Typography>
+      <Button onClick={() => window.open(previewUrl)} variant="contained">
```

Comment:
Does this work? It may be that `target` is not an allowed property for `Button`.

```suggestion
      <Button href={previewUrl} target="_blank" variant="contained">
```

Reply:
This makes a lot more sense, thank you!